content/cn/docs/concepts/configuration/secret.md

```diff
@@ -548,7 +548,7 @@ spec:
 
 ### 客户端使用 Secret API
 
-当部署与 secret API 交互的应用程序时，应使用诸如 [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) 之类的 [授权策略](https://kubernetes.io/docs/admin/authorization/) 来限制访问。
+当部署与 secret API 交互的应用程序时，应使用诸如 [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) 之类的 [授权策略](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) 来限制访问。
```

Comment:
We usually don't touch the translated pages because it may cause unwanted conflicts when the translators are merging their work back.

Reply:
Noted. I'll remove the changes for anything in cn across this and the other outstanding PRs I have that touch that area.